### PR TITLE
[Docs] Migrate "Phases of Terraform Adoption" from tutorials to docs

### PR DESCRIPTION
**Pull request by BrianMMcClain**
Currently the "[Phases of Terraform Adoption](https://developer.hashicorp.com/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity)" content lives in the tutorials as part of the Well-Architected Framework. We are currently in the process of migrating this content to the respective documentation.

### CODE REVIEW

File: website/docs/language/style.mdx [hunk 1]
```diff
@@ -678,4 +678,4 @@ For more information, refer to the [policy enforcement documentation](/terraform
 
 This article introduces some considerations to keep in mind as you standardize your organization's Terraform style guidelines. Enforcing a standard way of writing and organizing your Terraform code across your organization ensures that it is readable, maintainable, and shareable.
 
-The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity).
+The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity). To learn more Terraform adoption best practices, refer to the [Phases of Terraform adoption](/terraform/intro/phases) documentation.
```
Comment (rkoron007) [inline website/docs/language/style.mdx:681]:
```suggestion
The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity). To learn more Terraform adoption best practices, refer to [Phases of Terraform adoption](/terraform/intro/phases).
```

**Review by rkoron007 — APPROVED** [review]
Looks fantastic!

File: website/docs/language/style.mdx (hunk 1 above)
Comment (ritsok) [reply website/docs/language/style.mdx:681]: should the operational excellence link be removed? i thought that was the article we're relocating

Comment (BrianMMcClain) [reply website/docs/language/style.mdx:681]: Good catch, I'll get that removed!

**Pull request merged**